labman/db/util.py

```diff
@@ -0,0 +1,44 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2017-, labman development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from . import sql_connection
+from . import process
+
+
+def get_pools_listing():
+    """Generates pool listing for GUI with direct DB access for speed
+
+    Returns
+    -------
+    list of list
+    """
```

Comment:
Possible for a sentence or two of the constituents of the nested list? Looks like `(pool_composition_id, external_id, is_plate_pool, something)`. I don't know what the `is_plate_pool` or `something` are here though :/

Reply:
To be honest, neither do I ... would it suffice to add something like:
`Reproduces the functionality of [p.id, p.container.external_id, p.is_plate_pool, p.upstream_process.id] for p in PoolComposition.get_pools() but with direct calls to the database`?

Reply:
I'm good with that.

Reply:
k, added ...